support/SubmitScripts/Wheeler.sh

```diff
@@ -34,13 +35,18 @@ export SPECTRE_INPUT_FILE=./Input3DPeriodic.yaml
 echo "Running on the following nodes:"
 echo ${SLURM_NODELIST}
 
+CHARM_PPN=$(expr ${SLURM_CPUS_PER_TASK} - 1)
+echo "Slurm tasks: ${SLURM_NTASKS}"
+echo "Slurm cpus per task: ${SLURM_CPUS_PER_TASK}"
+echo "Charm ppn: ${CHARM_PPN}"
+
```

Comment:
Can you also add `module list`? I think it's helpful and should be in all the submit scripts so we know exactly what modules were used during the run.

Reply:
I haven't gotten that to work, which is why I didn't :/ But sure!

Reply:
okey dokes, done!